ambari/security/kerberos/enable-security.rst

```diff
@@ -0,0 +1,160 @@
+
+3. On Ambari Server and on each host in the cluster, add the unlimited security policy JCE jars to ``$JAVA_HOME/jre/lib/security/``.
+
+  For example, run the following to extract the policy jars into the JDK installed on your host:
+
+  ::
+
+    unzip -o -j -q jce_policy-8.zip -d /usr/jdk64/jdk1.8.0_60/jre/lib/security/
+
+4. Restart Ambari Server.
+
+5. Proceed to Running the Security Wizard.
+
+Running the Kerberos Security Wizard
+____________________________________
+
+Ambari provides three options for enabling Kerberos:
+
+* Existing MIT KDC
+* Existing Active Directory
+* Manage Kerberos principals and keytabs manually
+
+When choosing **Existing MIT KDC** or **Existing Active Directory**, the Kerberos Wizard prompts for information related to the KDC, the KDC Admin Account and the Service and Ambari principals.
+Once provided, Ambari will automatically create principals, generate keytabs and distribute keytabs to the hosts in the cluster.
+The services will be configured for Kerberos and the service components are restarted to authenticate against the KDC.
+This is the **Automated Setup** option.
+
+When choosing **Manage Kerberos principals and keytabs manually**, you must create the principals, generate and distribute the keytabs.
+Ambari will not do this automatically.
+This is the **Manual Setup** option.
+
+Launching the Kerberos Wizard (Automated Setup)
+_______________________________________________
+
+1. Be sure you have :doc:`Installed and Configured your KDC<kdc-config>` and have :ref:`prepared the JCE<ambari-install-jce>` on each host in the cluster.
+
+2. Log in to Ambari Web and Browse to ``Admin > Kerberos``.
+
+3. Click “Enable Kerberos” to launch the wizard.
+
+4. Select the type of KDC you are using and confirm you have met the prerequisites.
+
+5. Provide information about the KDC and admin account.
+
+  * In the **KDC** section, enter the following information:
+
+    * In the **KDC Host** field, the IP address or FQDN for the KDC host. Optionally a port number may be included.
+    * In the **Realm name** field, the default realm to use when creating service principals.
+    * (Optional) In the **Domains** field, provide a list of patterns to use to map hosts in the cluster to the appropriate realm. For example ``.yourdomain.local , yourdomain.local``.
+
+  * In the **Kadmin** section, enter the following information:
+
+    * In the **Kadmin** Host field, the IP address or FQDN for the KDC administrative host. Optionally a port number may be included.
+    * The **Admin principal** and **password** that will be used to create principals and keytabs.
+
+6. Modify any advanced Kerberos settings based on your environment.
+
+  * (Optional) To manage your Kerberos client krb5.conf manually (and not have Ambari manage the krb5.conf), expand the **Advanced krb5-conf** section and uncheck the "Manage" option. **You must have the krb5.conf configured on each host**.
```

Comment:
Put optional in bold and without parenthesis to alert better the user that all these options are..optionals.